docs/analysis/index.rst

```diff
@@ -0,0 +1,266 @@
+.. include:: ../references.txt
+
+.. _HLI:
```

Comment:
`HLI` -> `analysis` (and same for links below), to follow our label style in Gammapy.